.travis.yml

```diff
@@ -35,6 +35,7 @@ addons:
       - libtalloc-dev
       - zlib1g-dev
       - libgmime-2.6-dev
+      - libassuan-dev
```

Comment:
Is it a dependency of notmuch or gpg?  Se the comment above.,

Reply:
Doesn't seem happy with libgpgme 1.6: https://travis-ci.org/pazz/alot/jobs/411043268